src/utils/ConfigUtility.ts

```diff
@@ -0,0 +1,72 @@
+import fs from 'fs';
+
+const CONFIG_DIR: string = `${process.env.HOME}/.mcli`
```

Comment:
Where does this place the config file in relation to the final executable file?

Reply:
It will always place the file in .mcli in the home directory